tests/scripts/deploy_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
The scripts should be named specifically for the admission controller. For example, this script could be deploy_admission_controller_test.sh and the other one (deploy.sh) could be deploy_admission_controller.sh

Reply:
Changed the file name.